.flake8

```diff
@@ -0,0 +1,15 @@
+[flake8]
+ignore = 
+    E731,
+    W503,
+
+exclude = 
+    .git,
+    .tox,
+    __pycache__,
+    gulp,
+    requirements,
+    node_modules,
+    site,
```

Comment:
What's this `site` exclusion for?

Reply:
Brought over from cfgov-refresh because of mkdocs — for this and the jinja2 exclusion below (both can be removed) the original intent was to simply avoid unnecessary directory traversals looking for relevant files. In cfgov-refresh that added a noticeable amount of time.

Reply:
Oh, right, I forgot about mkdocs. Thanks for the explanation.